.github/workflows/binary-release.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32
         with:
-          go-version: "1.21"
+          go-version-file: 'go.mod'
```

Comment:
`stable` and `oldstable` aliases are a thing if you want to just stick to latest

Reply:
Going to stick with go-version-file to keep the version centralized. Although this is less important here.